arena-artifacts/Chart.yaml

```diff
@@ -59,3 +59,8 @@ dependencies:
     version: 0.1.0
     repository: "@gpu-exporter"
     condition: exporter.enabled,global.exporter.enabled
+  - name: job-supervisor
```

Comment:
job-supervisor or training-job-supervisor?

Reply:
job-supervisor will handle all kind of job, e.g. inference task, not only trainingjob